### DISCUSSION TITLE
blog/2024-11-29-23-ci-chong-qi/

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#69] [Announcements]
# blog/2024-11-29-23-ci-chong-qi/

被继妹掠夺美貌之后，我嫁给了一个自闭症老公。 系统说，只要老公能亲我一下，我就能恢复 1% 的美貌。 后来我的嘴都被这人亲烂了！

https://yanxuan.vercel.app/blog/2024-11-29-23-ci-chong-qi/

**Comment by nyr-github** [comment]
非常好看
